### DISCUSSION TITLE
global settings

### DISCUSSION THREAD

**Discussion by 123dpd** [PaulWoitaschek/Voice#1989] [Ideas]
Global settings.
I would like to be able to place settings like skip silence or speed of 1.5 globally.
Once set it would apply to any and all files automatically unless I manually change the setting for that file.
Further it would remain a global setting on restart.

**Comment by WayneShao** [comment]
I also really, really need this feature.

**Comment by ndas108** [comment]
Related to default playback speed feature request: #1790
